src/SB/Game/zTalkBox.h

```diff
@@ -150,9 +150,28 @@ namespace
     struct state_type
     {
         state_enum type;
+        
+        state_type(state_enum t);
+        virtual void start();
+        virtual void stop();
+    };
+
+    struct next_state_type
+    {
+        void stop();
+    };
 
+    struct start_state_type
+    {
+        void stop();
+        signed char update(xScene&, float);
```

Comment:
Although these arguments aren't used and thus don't need to have parameter names for referencing inside the function, for these update functions I would like us to explicitly include parameter names, as they are the most likely to be updated by modders in the future and we've already seen what these arguments are named even if they were stripped from the DWARF data. 

Please update the signature to this:

```suggestion
        S8 update(xScene& scn, F32 dt);
```